### ISSUE TITLE
mdmctl config set always prepends https

### ISSUE THREAD

**Issue by groob** [micromdm/micromdm#239]
```
mdmctl config set -api-token='secret' -server-url=http://localhost:8080

mdmctl config print
{
  "api_token": "secret",
  "server_url": "https://http:/",
  "skip_verify": false
}
```

The issue is somewhere in this block:
https://github.com/micromdm/micromdm/blob/aacac399459280578d891edf3f4cfce9f34f5ef4/cmd/mdmctl/config.go#L99-L110

**Comment by natewalck** [comment]
Damn it, this isn't fixed. I realized it has it hardcoded to https just after the if statement.

**Comment by groob** [comment]
It was fixed in #266
